### PR TITLE
Airport Challenge Joe Zhou

### PR DESCRIPTION
**Pull request by chn-challenger**
Figured best do a PR early to stop myself living in my own cave.

### CODE REVIEW

**Comment by chn-challenger** [comment]
What is this one hound violation...it does not say!

File: spec/plane_spec.rb [hunk 1]
```diff
@@ -1,30 +1,50 @@
...
+	  it 'is landed/not-flying after landing and set current airport' do
+  	 	class Airport; end
+  	 	airport = double(:airport_object)
+ 	  	allow(airport).to receive(:is_a?).with(Airport).and_return(true)
+			allow(airport).to receive(:land_permission).and_return(nil)
+			allow(airport).to receive(:plane_landed).with(subject).and_return(nil)
+			subject.land(airport)
+			expect(subject.flying).to eq(false)
+			expect(subject.current_airport).to eq(airport)
+		end
```
Comment (nikeshashar) [inline spec/plane_spec.rb]: @chn-challenger alignment issues on line 48 :)

Comment (chn-challenger) [inline spec/plane_spec.rb]: Thanks, Nikesh I just got the email from Hound as well.  Strangely I am not seeing the alignments like that on sublime.  Will investigate further.

PS: You have weekend challenge too?  Up so late! :)

Thanks again!

**Comment by sanjsanj** [comment]
Awesome :checkered_flag:

**Comment by chn-challenger** [comment]
Still need to fix the indents thanks to Sublime, please bear with me, getting pulled away by battleship...
to do list:
1)  battleship
2)  see my wife and son???
3)  fix these bloody indents :)

**Pull request closed without merging**